tests/testthat/test-unite.R

```diff
@@ -71,3 +71,14 @@ test_that("validates its inputs", {
     unite(df, "z", x:y, na.rm = 1)
   })
 })
+
+test_that("unite works for empty selections", {
```

Comment:
```suggestion
test_that("returns an empty string column (the initial value for the reduction algorithm) for empty selections (#1548)", {
```

This is a bit technical but it will help us if we question this in the future